src/legacy/core_plugins/kibana/public/home/components/add_data.js

```diff
@@ -44,46 +44,59 @@ const basePath = chrome.getBasePath();
 const AddDataUi = ({ apmUiEnabled, isNewKibanaInstance, intl, mlEnabled }) => {
 
   const renderCards = () => {
-    const apmTitle = intl.formatMessage({
-      id: 'kbn.home.addData.apm.nameTitle', defaultMessage: 'APM'
-    });
-    const apmDescription = intl.formatMessage({
-      id: 'kbn.home.addData.apm.nameDescription',
-      defaultMessage: 'APM automatically collects in-depth performance metrics and errors from inside your applications.'
-    });
-    const loggingTitle = intl.formatMessage({
-      id: 'kbn.home.addData.logging.nameTitle', defaultMessage: 'Logging'
-    });
-    const loggingDescription = intl.formatMessage({
-      id: 'kbn.home.addData.logging.nameDescription',
-      defaultMessage: 'Ingest logs from popular data sources and easily visualize in preconfigured dashboards.'
-    });
-    const metricsTitle = intl.formatMessage({
-      id: 'kbn.home.addData.metrics.nameTitle', defaultMessage: 'Metrics'
-    });
-    const metricsDescription = intl.formatMessage({
-      id: 'kbn.home.addData.metrics.nameDescription',
-      defaultMessage: 'Collect metrics from the operating system and services running on your servers.'
-    });
-    const securityTitle = intl.formatMessage({
-      id: 'kbn.home.addData.security.nameTitle', defaultMessage: 'Security analytics'
-    });
-    const securityDescription = intl.formatMessage({
-      id: 'kbn.home.addData.security.nameDescription',
-      defaultMessage: 'Centralize security events for interactive investigation in ready-to-go visualizations.'
-    });
+    const ampData = {
+      title: intl.formatMessage({
+        id: 'kbn.home.addData.apm.nameTitle', defaultMessage: 'APM'
+      }),
+      description: intl.formatMessage({
+        id: 'kbn.home.addData.apm.nameDescription',
+        defaultMessage: 'APM automatically collects in-depth performance metrics and errors from inside your applications.'
+      }),
+      ariaLabel: 'aria-describedby.addAmpButtonLabel'
```

Comment:
I'd rename `ariaLabel` property to `ariaDescribedby`, since `label` assumes a text, while we put an id.

```suggestion
      ariaDescribedby: 'aria-describedby.addAmpButtonLabel'
```